book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I think the rentals variable can also be modified: `attr_accessor :title, :author, :rentals`. Let's fix this.

Reply:
Will do, thank you

Reply:
Resolved 👍🏻